app/src/main/res/layout-land/story_chapter_view.xml

```diff
@@ -58,9 +64,9 @@
             android:layout_marginTop="4dp"
             android:layout_marginEnd="8dp"
             android:ellipsize="end"
-            android:fontFamily="sans-serif-medium"
+            android:fontFamily="sans-serif"
```

Comment:
Is this change according to the [mock](https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/cce938d6-e079-4761-bac0-8bde22ac6348/).

Reply:
no, the mock uses Roboto font, but I used sans-serif as it was being used earlier. Can I change it to Roboto?